src/main/java/com/roma/librarymanagment/controller/BookController.java

```diff
@@ -84,16 +84,16 @@ private String saveBook(Model model, String isbn, Publisher publisher,Author aut
     @RequestMapping(path = "/deleteBook/{id}", method = RequestMethod.GET)
     private String deleteBook(@PathVariable Long id){
         bookService.deleteById(id);
-        return "menu";
+        return "redirect:/books";
```

Comment:
Create a configuration properties class and add all this " "redirect:/books";

Reply:
Can you please clarify or point me to sites where I can clarity on what you requesting.

Reply:
https://www.baeldung.com/configuration-properties-in-spring-boot